code/notificationbuilder/src/main/java/com/adobe/marketing/mobile/notificationbuilder/PushTemplateIntentConstants.kt

```diff
@@ -23,48 +23,8 @@ object PushTemplateIntentConstants {
 
     object IntentKeys {
         const val CENTER_IMAGE_INDEX = "centerImageIndex"
-        const val IMAGE_URI = "imageUri"
         const val IMAGE_URLS = "imageUrls"
         const val IMAGE_CAPTIONS = "imageCaptions"
         const val IMAGE_CLICK_ACTIONS = "imageClickActions"
```

Comment:
Curious  why these are public. How would the caller use/need these keys?

Reply:
Also can we use the equivalent adb key, `adb_image` instead, as we do incase of title, body etc?

Reply:
Probably we only need Intent ACTION NAME to be exposed and not these keys. 
Will solve this in my next PR